style.css

```diff
@@ -0,0 +1,122 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  line-height: 1;
+  letter-spacing: 1.2;
+}
+
+html {
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  width: 100vw;
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.header-section {
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+  max-width: 100vw;
+  padding-bottom: 40px;
+  background-color: #fff;
+  border-bottom-left-radius: 8rem;
+  background-image: url(./images/header-img.png);
```

Comment:
- [ ] Please 🙏🏽 correct the background image to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219333322-8f3e2cd9-eb12-4756-b9af-51fc141246c8.png) | ![image](https://user-images.githubusercontent.com/58642949/219333465-3a8cae06-4be6-4c19-b635-f79a6771f43e.png) |